core/src/main/java/tc/oc/pgm/controlpoint/ControlPoint.java

```diff
@@ -403,37 +403,39 @@ private void dominateAndFireEvents(@Nullable Competitor dominantTeam, Duration d
    *
    * <p>If there is no neutral state, then the point is always either being captured by a specific
    * team, or not being captured at all.
-   *
-   * <p>If incremental capturing is disabled, then capturingTimeMillis is reset to zero whenever it
-   * stops increasing.
    */
   private void dominate(Competitor dominantTeam, Duration dominantTime) {
     if (!this.capturable || !TimeUtils.isLongerThan(dominantTime, Duration.ZERO)) {
       return;
     }
 
     ControlPointDefinition definition = this.getDefinition();
-
     if (this.controllingTeam != null && definition.hasNeutralState()) {
       // Point is owned and must go through the neutral state before another team can capture it
       if (dominantTeam == this.controllingTeam) {
-        this.regressCapture(dominantTeam, dominantTime);
+        // owner is recovering the point
+        recover(dominantTeam, dominantTime);
       } else if (dominantTeam != null) {
-        this.progressUncapture(dominantTeam, dominantTime);
-      } else if (!definition.isIncrementalCapture()) {
-        // No team is dominant and point is not incremental, so reset the time
-        this.capturingTime = Duration.ZERO;
+        // non-owner is uncapturing the point
+        uncapture(dominantTeam, dominantTime);
+      } else if (definition.getOwnedDecayRate() > 0.0) {
```

Comment:
```suggestion
      } else if (definition.getOwnedDecayRate() > 0) {
```
I'm pretty sure 0.0 is not required here